mips/malta.c

```diff
@@ -125,30 +125,15 @@ extern intptr_t parse_rd_start(const char *s);
 static void pm_bootstrap(unsigned memsize) {
   pm_init();
 
-  intptr_t rd_start = ramdisk_get_start();
-  unsigned rd_size = align(ramdisk_get_size(), PAGESIZE);
-
-  /*
-   * Ramdisk start address is expected to be page aligned and placed:
-   * - Directly after kernel's .bss section in case of OVPSim
-   * - One page after kernel's .bss section in case of Qemu
-   */
-  assert(is_aligned(rd_start, PAGESIZE));
-  assert(is_aligned(rd_start + rd_size, PAGESIZE));
-  assert(rd_start == 0 || (intptr_t)__kernel_end <= rd_start);
-
-  intptr_t real_kernel_end =
-    (rd_start == 0) ? (intptr_t)(__kernel_end) : (intptr_t)(rd_start + rd_size);
-
-  pm_seg_t *seg = (pm_seg_t *)real_kernel_end;
+  pm_seg_t *seg = (pm_seg_t *)__kernel_end;
```

Comment:
I understand you'll change that to use `kernel_sbrk` with next PR. Is that correct?

Reply:
That's correct.